src/Exceptions/InvalidDumperException.php

```diff
@@ -0,0 +1,24 @@
+<?php namespace Geocoder\Laravel\Exceptions;
```

Comment:
I guess what's missing here is the license comment, that's why its not on separate lines. I'll add the license comment and place the namespace below that.